docs/resources/check_group.md

```diff
@@ -67,10 +77,6 @@ resource "checkly_check_group" "test_group1" {
       failed_run_threshold = 1
     }
 
-    time_based_escalation {
```

Comment:
Since the `RUN_BASED` escalation policy is used, we shouldn't add a `time_based_escalation` policy. Otherwise there will always be a diff in `terraform apply`.